sources/plugins/LametricTime/locales/fr.json

```diff
@@ -0,0 +1,32 @@
+﻿{
+  "name": "LametricTime",
+  "description": "Support de l'équipement Lametric",
```

Comment:
Prefer "Support de l'équipement Lametric Time", and add a link to Lametric Time official page (https://lametric.com/en-US/time/overview)